bindata/manifests/sriov-config-service/kubernetes/sriov-config-service.yaml

```diff
@@ -7,7 +7,7 @@ contents: |
 
   [Service]
   Type=oneshot
-  ExecStart=/var/lib/sriov/sriov-network-config-daemon -v 2 --zap-log-level 2 service --phase pre
+  ExecStart=/var/lib/sriov/sriov-network-config-daemon service --phase pre
```

Comment:
this means reboot on update ? since systemfile changes

Reply:
yes it will make a reboot one time